build-llvm.py

```diff
@@ -315,6 +322,28 @@ def cleanup(build_folder, incremental):
     build_folder.mkdir(parents=True, exist_ok=True)
 
 
+def get_final_stage(args):
+    """
+    Gets the final stage number, which depends on PGO
+    :param args: The args variable generated by parse_parameters
+    :return: The final stage number
+    """
+    if args.pgo:
+        return 3
+    else:
+        return 2
+
+
+def instrumented_stage(args, stage):
+    """
+    Returns true if we are using PGO and on stage 2
+    :param args: The args variable generated by parse_parameters
```

Comment:
Maybe just pass `args.pgo` as a boolean?

Reply:
I think it is easier to have the `args.pgo` in here, rather than in all of the call spots but I don't feel strongly about it.

Reply:
Yeah I don't either, let's just keep `args`.